scripts/change-extensibility.js

```diff
@@ -0,0 +1,76 @@
+if(!process.env.npm_config_plugin) {
+    throw new Error("Missing plugin name.");
+}
+
+let pluginSpaceName = process.env.npm_config_plugin;
+let metadata = process.env.npm_config_name;
+let pluginForgeVersion = process.env.npm_config_forge
+let mabsMin = process.env.npm_config_mabs;
+
+let repository = process.env.npm_config_repository;
+let branch = process.env.npm_config_branch;
+
+let environment = process.env.npm_config_environment;
+
+if(process.env.npm_config_authentication == null) {
+    throw new Error("Missing authentication argument.");
+}
+let basicAuthentication = process.env.npm_config_authentication;
+
+let url = "https://" + environment + "/CodeUpdater/rest/Bulk/ExtensabilityUpdate";
+
+let extensibilityChangeJson = {
+    plugin :{
+        url: `https://${repository}#${branch}`,
+    },
+    metadata: { 
+        "mabs-min": mabsMin ? mabsMin : "9.0.0",
```

Comment:
Considering that v9 is already deprecated, should the default value be `10.0.0`?

Reply:
indeed!